frontend/gateway/container.go

```diff
@@ -293,6 +293,7 @@ type gatewayContainer struct {
 	cancel     func()
 }
 
+//nolint:staticcheck // SA4009 ctx is overwritten by design
```

Comment:
We can avoid this by explicitly giving the arg the name `_`, and then later creating it with `ctx :=`.